language/message/de.po

```diff
@@ -211,6 +211,9 @@ msgstr "Friedhof"
 msgid "beauty salon"
 msgstr "Schönheitssalon"
 
+msgid "beekeeper"
+msgstr "beekeeper"
```

Comment:
I'm not sure about capitalization in German, most of the diff seems to be capitalized, but not these two?